src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -24,6 +24,7 @@ class ParameterBag implements ParameterBagInterface
 {
     protected $parameters = array();
     protected $resolved = false;
+    private $resolver;
```

Comment:
Should we expose this resolver with a getter and deprecate ``resolveValue`` and ``resolveString`` in the ``ParameterBag``?